R/runtime-caches.R

```diff
@@ -0,0 +1,72 @@
+#' Get runtime caches
+#' 
+#' View the runtime caches on a Connect server.
+#' 
+#' @param client A `Connect` object.
+#' 
+#' @returns A tibble of runtime caches on the server, showing `language`,
+#' `version` and `image_name`. For Connect servers not using off-host execution,
+#' `image_name` is `"Local"`.
+#' 
+#' @examples
+#' \dontrun{
+#' client <- connect()
+#' get_runtime_caches(client)
+#' }
+#' 
+#' @family server management functions
+#' @export
+get_runtime_caches <- function(client) {
+  validate_R6_class(client, "Connect")
+  res <- client$GET(v1_url("system/caches/runtime"))
+  purrr::map_dfr(res$caches, ~.x)
+}
+
+#' Delete a runtime cache
```

Comment:
Do we need to include any information regarding permissions in the documentation?

Reply:
Good point, I'll add a note to that effect.